pkg/registry/apis/provisioning/jobs/pullrequest/worker.go

```diff
@@ -75,8 +79,7 @@ func (c *PullRequestWorker) Process(ctx context.Context,
 
 	progress.SetMessage(ctx, "listing pull request files")
 	base := cfg.GitHub.Branch
-	ref := options.Hash
-	files, err := prRepo.CompareFiles(ctx, base, ref)
+	files, err := prRepo.CompareFiles(ctx, base, options.Ref)
```

Comment:
not sure about this change. I remember the the Hash was there for a reason. One of the repository functions needed the hash and not the branch. 

If hash is not used here, it's probably not used anymore and we should remove it.